packages/framework-tools/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "@redwoodjs/framework-tools",
+  "version": "6.0.7",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/redwoodjs/redwood.git",
+    "directory": "packages/framework-tools"
+  },
+  "license": "MIT",
+  "type": "module",
+  "exports": {
+    "types": "./dist/buildDefaults.d.ts",
+    "import": "./dist/buildDefaults.mjs",
+    "default": "./dist/buildDefaults.cjs"
+  },
```

I'm  not sure that we need this to be a dual package unless you had reasons otherwise?

```suggestion
  "exports": {
    "types": "./dist/buildDefaults.d.ts",
    "default": "./dist/buildDefaults.mjs"
  },
```

We'd also simplify the `./build.ts` file in this package

---

No, esm is enough